src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -376,6 +387,10 @@ function FloatingActionButtonAndPopover(
                                   numberOfLinesDescription: 1,
                                   onSelected: () => interceptAnonymousUser(() => navigateToQuickAction()),
                                   shouldShowSubscriptRightAvatar: ReportUtils.isPolicyExpenseChat(quickActionReport),
+                                  shouldRenderTooltip: quickAction?.isFirstQuickAction,
+                                  shouldForceRenderingTooltipLeft: true,
+                                  renderTooltipContent: renderQuickActionTooltip,
```

Comment:
Looking at this call, I'm wondering what's the relation between the `shouldRenderTooltip` and `renderTooltipContent` properties.

```ts
    /** Render custom content inside the tooltip. */
    renderTooltipContent?: () => ReactNode;
```

`renderTooltipContent` is declared optional. Isn't it the case that `shouldRenderTooltip` is true if and only if `renderTooltipContent` is not `undefined`?

Couldn't we nuke `shouldRenderTooltip`, and pass `renderTooltipContent: quickAction?.isFirstQuickAction ? renderQuickActionTooltip : undefined`? Of course, adjusting all the other places accordingly.

I believe that, in a chain reaction, we could nuke this check...

```ts
function EducationalTooltip({shouldRender = true, children, ...props}: TooltipExtendedProps) {
    if (!shouldRender) { // <-- this check
        return children;
    }
    // ...
```

...and maybe even the whole `EducationalTooltip` / `BaseEducationalTooltip` in general.

Maybe you could use this helper local function in `MenuItem.tsx`. It's only an option.

```tsx
    function wrapWithTooltip(children: React.JSX.Element): React.JSX.Element {
        if (renderTooltipContent) {
            return (
                <EducationalTooltip
                    shouldRender={shouldRenderTooltip}
                    shouldForceRenderingLeft={shouldForceRenderingTooltipLeft}
                    renderTooltipContent={renderTooltipContent}
                    wrapperStyle={tooltipWrapperStyle}
                    shiftHorizontal={styles.popoverMenuItem.paddingHorizontal}
                    shiftVertical={styles.popoverMenuItem.paddingVertical / 2}
                >
                    {children}
                </EducationalTooltip>
            );
        } else {
            return children;
        }
    }
```

```tsx
            {wrapWithTooltip(
                <View>
                    <Hoverable>
                        {(isHovered) => (
                            <PressableWithSecondaryInteraction
                            // ...
```

Reply:
> We also have text beside renderTooltipContent

No, in `MenuItem` we don't.

Just to ensure that we're on the same page in the same context, I started an isolated focused thread in the context of the code we're discussing.